docs/source/reference/kubernetes/index.rst

```diff
@@ -0,0 +1,154 @@
+
+4. You can now run any SkyPilot task on your Kubernetes cluster.
+
+   .. code-block:: console
+
+        $ sky launch --cpus 2+
+        == Optimizer ==
+        Target: minimizing cost
+        Estimated cost: $0.0 / hour
+
+        Considered resources (1 node):
+        ---------------------------------------------------------------------------------------------------
+         CLOUD        INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN
+        ---------------------------------------------------------------------------------------------------
+         Kubernetes   2CPU--2GB         2       2         -              kubernetes    0.00          ✔
+         AWS          m6i.large         2       8         -              us-east-1     0.10
+         Azure        Standard_D2s_v5   2       8         -              eastus        0.10
+         GCP          n2-standard-2     2       8         -              us-central1   0.10
+         IBM          bx2-8x32          8       32        -              us-east       0.38
+         Lambda       gpu_1x_a10        30      200       A10:1          us-east-1     0.60
+        ---------------------------------------------------------------------------------------------------.
+
+
+.. note::
+  SkyPilot will use the cluster and namespace set in the ``current-context`` in the
+  kubeconfig file. To manage your ``current-context``:
+
+  .. code-block:: console
+
+    $ # See current context
+    $ kubectl config current-context
+
+    $ # Switch current-context
+    $ kubectl config use-context mycontext
+
+    $ # Set a specific namespace to be used in the current-context
+    $ kubectl config set-context --current --namespace=mynamespace
+
+FAQs
+----
+
+* **Are autoscaling Kubernetes clusters supported?**
+
+  Yes - however they currently require adjusting the resource provisioning timeout (:code:`Kubernetes.TIMEOUT` in `clouds/kubernetes.py`) to a large value to give enough time for the cluster to autoscale. We are working on a better interface to adjust this timeout - stay tuned!
+
+* **What container image is used for tasks? Can I specify my own image?**
+
+  We use and maintain a SkyPilot container image that has conda and a few other basic tools installed. You can specify a custom image to use in `clouds/kubernetes.py`, but it must have rsync, conda and OpenSSH server installed. We are working on a interface to allow specifying custom images through the :code:`image_id` field in the task YAML - stay tuned!
+
+* **Can SkyPilot provision a Kubernetes cluster for me? Will SkyPilot add more nodes to my Kubernetes clusters?**
+
+  The goal of SkyPilot on Kubernetes is to run SkyPilot tasks on resources in an existing Kubernetes cluster. It does not provision any new Kubernetes clusters or add new nodes to an existing Kubernetes cluster. The Kubernetes control plane remains untouched.
```

Comment:
```suggestion
  The goal of Kubernetes support is to run SkyPilot tasks in an existing Kubernetes cluster. It does not provision any new Kubernetes clusters or add new nodes to an existing Kubernetes cluster. The Kubernetes control plane remains untouched.
```
"The Kubernetes control plane remains untouched." this part reads a bit surprising under this Q.